src/transactions/TransactionFrame.h

```diff
@@ -100,20 +100,19 @@ class TransactionFrame : public TransactionFrameBase
                               uint64_t lowerBoundCloseTimeOffset,
                               uint64_t upperBoundCloseTimeOffset,
                               std::optional<FeePair> sorobanResourceFee,
-                              MutableTxResultPtr txResult) const;
+                              MutableTxResultPtr txResult,
+                              EventManagerPtr evtManager) const;
 
     virtual bool isBadSeq(LedgerHeaderWrapper const& header,
                           int64_t seqNum) const;
 
-    ValidationType commonValid(AppConnector& app,
-                               std::optional<SorobanNetworkConfig> const& cfg,
-                               SignatureChecker& signatureChecker,
-                               LedgerSnapshot const& ls, SequenceNumber current,
-                               bool applying, bool chargeFee,
-                               uint64_t lowerBoundCloseTimeOffset,
-                               uint64_t upperBoundCloseTimeOffset,
-                               std::optional<FeePair> sorobanResourceFee,
-                               MutableTxResultPtr txResult) const;
+    ValidationType commonValid(
+        AppConnector& app, std::optional<SorobanNetworkConfig> const& cfg,
+        SignatureChecker& signatureChecker, LedgerSnapshot const& ls,
+        SequenceNumber current, bool applying, bool chargeFee,
+        uint64_t lowerBoundCloseTimeOffset, uint64_t upperBoundCloseTimeOffset,
+        std::optional<FeePair> sorobanResourceFee, MutableTxResultPtr txResult,
+        EventManagerPtr evtManager) const;
```

Comment:
nit: Could you please use `eventManager` everywhere for consistency? `evt` saves just two characters and is less readable.

Reply:
fixed